app/lib/features/account_deletion_config.rb

```diff
@@ -29,14 +34,14 @@ def initialize(config = {})
     private
 
     def format_config(config)
-      config = (config || {}).each_with_object({}) do |(key, value), collection|
-        collection[key.to_sym] = value if value.is_a?(Integer)
+      config = (config || {}).symbolize_keys.slice(*EXPECTED_VALUES.keys).each_with_object({}) do |(key, value), collection|
```

Comment:
what does `slice` with has values? can you explain why we need these `EXPECTED_VALUES` ?

Reply:
@macejmic @guicassolato 
When this feature was introduced, there was a big debate about what to do if someone introduces wrong values in the config. It all started in https://github.com/3scale/porta/pull/574#discussion_r253816442
And later we kept talking between these options if the user introduces wrong config:
- Not booting at all
- Booting without this feature enabled and just showing a warning informing of that
- Let it raise errors (for example if it is not an integer and it does `days.ago` over it) when it is going to be used.

And back then we decided to do the 2nd one. To do the 2nd one, this is the way to do it 🤷‍♀️ If you have a better idea, please elaborate about it 😝

Reply:
The best way to do it IMO is "If the configs are there, they should be considered valid."

Reply:
Ok, spoken in the team, it is decided: "If the configs are there, they should be considered valid."